src/Prefixer.js

```diff
@@ -90,7 +92,7 @@ export function prefix (value, length) {
 				break
 		// display: (flex|inline-flex|inline-box)
 		case 6444:
-			switch (charat(value, strlen(value) - 2 - (indexof(value, '!important') && 10))) {
+			switch (charat(value, strlen(value) - 2 - (~indexof(value, '!important') && 10))) {
```

Comment:
Does this work, i.e ~10 is not 10 but -11, was this breaking as it where?

Reply:
Yes, it wasn't working. We want to subtract 10 if there is `!important` but `-1 /*indexof*/ && 10` is 10 instead of 0, `~` fixes this, it will evaluate to 10 for all return values different than -1

Reply:
Oh, my bad, i misunderstood.